.github/workflows/app-etterlatte-saksbehandling-ui.yaml

```diff
@@ -72,13 +75,16 @@ jobs:
         working-directory: apps/${{ env.APP_NAME }}/client
         run: CI=true yarn test
       - name: Build and publish Docker image
-        env:
-          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
-        run: |
-          docker login ghcr.io -u ${GITHUB_REPOSITORY} -p ${GITHUB_TOKEN}
-          docker build --tag ${{ env.IMAGE }} --tag ${{ env.branch-image }}:${{ env.GITHUB_REF_SLUG }} apps/${{ env.APP_NAME }}
-          docker push ${{ env.IMAGE }}
-          docker push ${{ env.branch-image }}:${{ env.GITHUB_REF_SLUG }}
+        uses: nais/docker-build-push@v0
+        id: docker-build-push
+        with:
+          team: etterlatte
+          identity_provider: ${{ secrets.NAIS_WORKLOAD_IDENTITY_PROVIDER }}
+          project_id: ${{ vars.NAIS_MANAGEMENT_PROJECT_ID }}
+          dockerfile: apps/${{ env.APP_NAME }}/Dockerfile
+          docker_context: apps/${{ env.APP_NAME }}/
+          image_suffix: ${{ github.workflow }}
+          tag: ${{ env.GITHUB_REF_SLUG }}
 
```

Comment:
Er det noe nytteverdi i å ha en tilsvarende print docker tag step (GITHUB_STEP_SUMMARY) som i .build.yaml ?

Reply:
Usikker. Har ikkje kjent på behovet for det så langt, så vil tippe nei.